docs-website/src/pages/_components/Logos/index.js

```diff
@@ -81,11 +81,11 @@ const companiesByIndustry = [
   {
     name: "Financial & Fintech",
     companies: [
-      {
-        name: "Saxo Bank",
-        imageUrl: "/img/logos/companies/saxobank.svg",
-        imageSize: "default",
-      },
+      // {
+      //   name: "Saxo Bank",
+      //   imageUrl: "/img/logos/companies/saxobank.svg",
+      //   imageSize: "default",
+      // },
```

Comment:
@jayacryl i think this is outdated & results in conflict ; you should make a change in the file https://github.com/datahub-project/datahub/blob/master/docs-website/adoptionStoriesIndexes.json

note that if you comment out saxo bank in this file it will remove both the logo and the case study card. 
lmk if you want to do it differently - we can add a config on the logo carousel on only pulling logos that has imgUrl , etc. in that way we can just comment out the imgurl part and then the case study card will still visible